src/components/ConfirmTermination/ConfirmTerminationStyles.ts

```diff
@@ -0,0 +1,19 @@
+import styled from 'styled-components';
+import { transparentize } from 'polished';
+import { themeColor } from '@amsterdam/asc-ui';
+
+export interface Props {
+	backdropOpacity?: number;
+	zIndexOffset?: number;
+}
+
+export const BackDropStyle = styled.div<Props>`
+	width: 100%;
+	height: 100%;
+	top: 0;
+	left: 0;
+	position: absolute;
+	z-index: ${({ zIndexOffset }) => (zIndexOffset ? 21 + zIndexOffset : 21)};
```

Comment:
This seems rather arbitrary; why not just set the zIndex value from the prop?

Reply:
The default z-index for the ASC Modal is 20. 21 puts it on top of another modal by default. The zIndexOffset gives the user more control over the z-postion of a confirmTermination modal.

Reply:
Worth adding a comment explaining that?

Reply:
Added comment :)